### PR TITLE
chore: add coming soon modules

### PR DESCRIPTION
**Pull request by NathanFlurry**

### CODE REVIEW

**Comment by NathanFlurry** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb-modules/120?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#120** <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb-modules/120?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#110** <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb-modules/110?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116** <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb-modules/116?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @NathanFlurry and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: modules/matchmaker/module.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Matchmaker",
+  "description": "Ticket-based matchmaking that balances skill, latency, and more. (Coming mid-2024)",
```
Comment (Blckbrry-Pi) [inline modules/matchmaker/module.json:3]: This is a lot to promise. Do we already have the code for this?

**Review by Blckbrry-Pi — APPROVED** [review]
Looks good!

**Comment by NathanFlurry** [comment]
### Merge activity

* **Aug 16, 8:51 AM EDT**: @NathanFlurry started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/rivet-gg/opengb-modules/120).

**Pull request merged**